package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "typeorm",
   "private": true,
-  "version": "0.0.9",
+  "version": "0.1.0-alpha.1",
```

Comment:
Why no SemVer? I understand the `-alpha.x` suffix but switching from 0.0.9 to 0.1.0 is MINOR release, so no breaking changes, just new optional features. So no removing `Table` decorators or changing from `getResults` to `getMany` between 0.0.2 and 0.0.3. It's really bad that users has to save excact version of TypeORM to avoid this kind of changes because they can't rely on semver.
Why are you afraid of releasing 1.0.0? There always will be stuffs to improve or change or add, so you can't stay in 0.x.x forever 😉